src/ValueFormatters/TimeFormatter.php

```diff
@@ -64,14 +70,87 @@ public function format( $value ) {
 			throw new InvalidArgumentException( 'Data value type mismatch. Expected a TimeValue.' );
 		}
 
-		$formatted = $value->getTime();
+		$formatted = $this->getFormattedTimestamp( $value );
+		// FIXME: Temporarily disabled.
+		//$formatted .= ' (' . $this->getFormattedCalendarModel( $value->getCalendarModel() ) . ')';
```

Comment:
I'm not sure. We removed this temporarily to have less confusing information in diffs. If we all agree what the calendar model means, and we all agree that this consensus should be visible to our users, then yes, this line should be enabled. On the other hand: This formatter is currently not used. We use the HTML formatter instead.

I would like to discuss this in an other pull request and merge this as it is. This plain text formatter is primarily meant as a fall back for external users of our libraries.